Prosjektrapport-V2.md

```diff
@@ -112,7 +112,63 @@ Det er viktig å dokumentere ikke bare alle hensyn som tas, men også hvilke hen
+Ved å integrere UseCase og analysere bruksmønstre, kan utviklingsteam utvikle mer intuitive og brukervennlige applikasjoner og systemer som treffsikkert adresserer målgruppens behov. 
+
+
 ### Aktører og UseCase(UC)
+Når vi utvikler eller analyserer systemer, spesielt innen programvareutvikling, blir to konsepter ofte fremhevet for å forstå og definere hvordan systemet skal brukes: Aktører og UseCase. Disse konseptene er avgjørende for å identifisere funksjonelle krav og forventede interaksjoner med systemet.
+
+#### Hva er Aktører?
+Aktører er entiteter som interagerer direkte med systemet. De kan være mennesker, andre systemer eller eksterne enheter. Hensikten med å identifisere aktører er å klargjøre hvem eller hva som vil bruke systemet, og på hvilke måter. Aktører spiller forskjellige roller i forhold til systemet. 
+
+Primære Aktører: De som initierer interaksjoner for å oppnå et mål. For eksempel, en bruker som søker informasjon. 
+
+Sekundære Aktører: De som støtter eller responderer i interaksjonene. Dette kan være eksterne systemer som systemet kommuniserer med for å utføre en oppgave.
+
+Offstage Aktører: De som har indirekte interesse i interaksjonen, men som ikke deltar direkte. Dette kan være regulatoriske myndigheter eller systemadministratorer.
+Å identifisere aktører gir grunnlag for å forstå systemets grensesnitt og hvordan forskjellige brukere vil interagere med det.
+
+
+
+##### Hva er UseCase?
+
+UseCase beskriver en sekvens av hendelser som utføres av systemet for å oppnå et spesifikt mål for en aktør. Det er en teknikk brukt for å fange opp funksjonelle krav og illustrerer systemets oppførsel under forskjellige forhold. UseCase er delt inn i flere komponenter:
+
+Navn: Gir en kort beskrivelse av handlingen eller målet.
+Aktør: Den som starter UseCase-scenariet.
+Scenario: En detaljert beskrivelse av hendelsene som utspiller seg, inkludert hovedflyt og alternative flyter.
+Forutsetninger: Betingelser som må være oppfylt før UseCase kan starte.
+Resultat: Tilstanden til systemet eller endringene som er oppnådd etter at UseCase er fullført.
+Bruken av UseCase hjelper teamet med å visualisere systemets funksjonalitet fra brukerens perspektiv og sikrer at alle brukerbehov er adressert.
+
+#### Samspillet Mellom Aktører og UseCase
+Samspillet mellom aktører og UseCase er grunnleggende i systemutvikling. Ved å kartlegge aktørene og definere tilhørende UseCase, skapes en brukersentrert tilnærming til systemdesign. Dette bidrar til å sikre at systemet leverer verdi til alle brukere og møter deres behov på en effektiv måte. Videre hjelper det utviklingsteamet med å identifisere grensesnittkrav, forstå brukerinteraksjoner og prioritere funksjonalitetsutvikling basert på brukerscenarier.
+
+I sin helhet gir integrasjonen av aktører og UseCase et sterkt rammeverk for å utforske, dokumentere og validere hvordan et system skal fungere, og spiller en kritisk rolle i et velfungerende system. 
```

Comment:
Jeg ville ha flyttet denne linjen opp til 168 egentlig. Vi trenger vel ikke et eget avsnitt bare for denne setningen?